README.md

```diff
@@ -284,6 +284,16 @@ Conventions:
 - use a \`.class\` selector for container properties
 - use a \`[itemprop=...]\` selector for singular properties, or items of container properties
 
+There are several additional selectors which are not found as the Stencila Schema definitions. These are:
+
+| Selector            | Description                                                                                                                                                                              | Target                                                                                                                        |
+| :------------------ | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :---------------------------------------------------------------------------------------------------------------------------- |
+| `:--root`           | It maps to the root element generated by Encoda. This is done to avoid potential clashes with external stylesheets, and to ensure that Thema only styles semantically annotated content. | `[data-itemscope=root]`                                                                                                       |
```

Comment:
Not sure what hard dependency there might be on the name from Encoda, but this is very similar to the `:root` CSS selector. Do you think it's perhaps worth renaming to `:--Root` to make it less similar?

Reply:
Hmmm, during development I did mistakenly type `:root` by accident once or twice, so do see the concern.
Not sure if it's an implementation detail or if the end-user of this selector would care, but the naming convention of TitleCase vs camelCase depends on if the selector targets a MicroData `itemtype` vs `itemprop`.
Do you think clarifying the fact that this selector is supposed to be used in place of `:root`, and adding a Stylelint warning would be enough to helping catch issues?

Reply:
I share the concern about similarity between `:--root` and `:root`. On one hand it is good because it signals the intent, but on the other, can cause confusion. Regardless of either, it is error prone.

I think Alex's suggestions are good. Let's try them and see if the issue arises again. It's not a painful change if we do need to rename in the future.